mne_bids/mne_bids.py

```diff
@@ -285,11 +306,22 @@ def _scans_tsv(raw, raw_fname, fname, verbose):
         Relative path to the raw data file.
     fname : str
         Filename to save the scans.tsv to.
+    write_mode : str, one of ('append', 'overwrite', 'error')
+        How the file should be handled if is exists already.
+        Defaults to 'error'.
+        If write_mode == 'append' the existing file will have the
+        new data added to it.
+        If a duplicate entry is found it will be replaced with the new values
+        provided.
+        If write_mode == 'overwrite' the existing file will be
+        removed and replaced with the new data.
+        If write_mode == 'error' an `OSError` will be raised if the data
+        already exists in the file.
     verbose : bool
         Set verbose output to true or false.
 
     """
-    # get MEASurement date from the data info
+    # get measurement date from the data info
```

Comment:
you just undid something @sappelhoff did on purpose :-P

Reply:
oh, I always assumed it was a typo... Why is it like that???
Will revert

Reply:
yeah, everybody assumes it to be a typo it seems ... it's there because it made me understand what the omnipresent `meas_date` in MNE-Python is an abbreviation for. Took me embarrassingly long to get it so I though I sneak in this little helper ;-)